src/components/events/single-event/CommentsCard.js

```diff
@@ -25,12 +55,52 @@ const ParticipantCard = (props) => {
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    setComments([...comments, newComment]);
+    setComments([
+      ...comments,
+      {
+        id: idCounter,
+        user_id: me.id,
+        event_id: props.eventId,
+        parent: -1,
+        root: idCounter,
+        date_created: new Date().getTime(),
+        description: newComment,
+      },
+    ]); //placeholder values for keys, will need to get from state or database
+    setIdCounter(idCounter + 1);
     setNewComment("");
+
+    // axiosWithAuth()({
+    //   url: `${process.env.REACT_APP_BASE_URL}/graphql`,
+    //   method: "post",
+    //   data: {
+    //     query: print(NEW_COMMENT),
+    //     variables: { comment: e.target.value },
+    //   },
+    // })
+    //   .then((res) => {
+    //     //add comments to state
+    //   })
+    //   .catch((err) => {
+    //     console.log(err.message);
+    //   });
   };
 
+  useEffect(() => {
+    if (comments) {
+      const sorted = comments.sort((a, b) => {
+        if (a.root === b.root) {
+          if (a.parent === b.parent) {
+            return a.date_created - b.date_created;
+          } else return a.parent - b.parent;
+        } else return a.root - b.root;
+      });
+      setOrganizedComments(sorted);
+    }
+  }, [comments]);
```

Comment:
my attempt to organize comment hierarchy.  Would love feedback if there is a better way to do this.